### DISCUSSION TITLE
.NET MAUI

### DISCUSSION THREAD

**Discussion by michael-duren** [spark-dotnet/framework#43] [Ideas]
Curious to what your thoughts would be on creating a .NET MAUI template. 
I've been interested in using .NET MAUI on personal projects but the template doesn't have hot reload available for Mac OS Use, so I haven't gotten into it.

**Comment by pratikghag1089** [comment]
I believe he will primarily focus on Blazor (web) given that he's developing https://dotnetengine.com/, which is centered on web technology. Additionally, for MAUI, a separate project might be necessary since it's a cross-platform framework.
